tests/test_client.py

```diff
@@ -819,46 +853,15 @@ def update_roles_v1(self, obj):
     def update_modifier_list(self, obj, version): # TODO try v1 endpoint in produciton env
         obj_id = obj.get('id')
         body = {'batches': [{'objects': [{'id': obj_id,
-                                         'type': 'MODIFIER_LIST',
+                                          'type': 'MODIFIER_LIST',
                                           'version': version,
                                           'modifier_list_data': {'name': self.make_id('modifier_list')}}]}],
                 'idempotency_key': str(uuid.uuid4())}
         return self.post_category(body)
 
-    def _update_items_by_modifier_list(self, obj_id):  # REFACTOR
-        # TODO use this as an update for the 'items' stream, not 'modifier_lists'
-
-        # Get all items
-        start_date = '2020-07-29T00:00:00Z'
-        items = self.get_all('items', start_date)
-
-        # Randomly select an item with an assigned modifier_list
-        items_with_mods = [i for i in items if i.get('item_data', {'modifier_list_info': False}).get('modifier_list_info')]
-        item = random.choice(items_with_mods)
-
-        # Set update base on existing data
-        item_data = item.get('item_data')
-        enabled = item_data.get('modifier_list_info')[0].get('enabled')
-        modifier_list_id = item_data.get('modifier_list_info')[0].get('modifier_list_id')
-        modifier = 'modifier_lists_to_disable' if enabled else 'modifier_lists_to_enable'
-
-        body = {'item_ids': [item.get('id')],
-                modifier: [modifier_list_id],
-        }
-
-        resp = self._client.catalog.update_item_modifier_lists(body)
-        if resp.is_error():
-            raise RuntimeError("Unable to UPDATE modifier_lists: {}".format(resp.errors))
-        # Updated modifier_lists return only the 'updated_at' in the response
-        # so we are grabbing the response body from a get to copare
-        mod_lists = self.get_all('modifier_lists', start_date)
-        updated_mod_lists = [ml for ml in mod_lists if ml.get('id') == modifier_list_id]
-        assert len(updated_mod_lists) == 1
-        return updated_mod_lists
-
```

Comment:
This shouldn't have been delete but I have it on my current branch so I think it's fine.

Reply:
This method is not used anywhere so I removed it.